### DISCUSSION TITLE
Web animations

### DISCUSSION THREAD

**Discussion by redbar0n** [software-mansion/react-native-reanimated#1908] [Q&A]
Does Reanimated use CSS Animations or the [Web Animations API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API) (WAAPI) or something else to run animations on the web?

WAAPI run with JS, but is apparently just as performant as CSS Animations, since they are offloaded from the main JS thread in the the browser to a separate thread: https://css-tricks.com/css-animations-vs-web-animations-api/

I couldn't find info about this in the docs, where I would have expected it:
https://docs.swmansion.com/react-native-reanimated/docs/web-support

@karol-bisztyga

**Comment by karol-bisztyga** [comment]
cc @piaskowyk
